WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailsContainerFragment.java

```diff
@@ -31,9 +35,7 @@
 
 import java.util.ArrayList;
 
-public class HistoryDetailFullScreenDialogFragment extends Fragment implements FullScreenDialogContent {
-    protected FullScreenDialogController mDialogController;
-
+public class HistoryDetailsContainerFragment extends Fragment {
```

Comment:
```suggestion
public class HistoryDetailContainerFragment extends Fragment {
```

Can we rename this class `HistoryDetailContainerFragment` for consistency?